keyboards/protokeeb/rev1/config.h

```diff
@@ -0,0 +1,50 @@
+/* Protokeeb Copyright 2024 A-Tech Officials (@atechofficials)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. 
+ * If not, see <https://www.gnu.org/licenses/gpl-3.0.html>.
+ */
+
+#pragma once
+
+// Define the pin connected to the encoder push button
+#define ENCODER_BUTTON_PIN GP14
+
+// Define the row and column for the encoder button
+#define ENCODER_BUTTON_ROW 0
+#define ENCODER_BUTTON_COL 4
+
+/* Peripheral used */
+#define WS2812_PIO_USE_PIO1
+
```

Comment:
```suggestion
```

Reply:
If I am not wrong the PIO driver is used by RP2040 to control the WS2812 RGB LEDs as mentioned here:
https://docs.qmk.fm/drivers/ws2812#pio-driver
https://docs.qmk.fm/drivers/ws2812#arm-pio-driver
Additionally, this line of code instructs the firmware compiler to use the PIO1 peripheral instead of PIO0 for ARM-based MCUs like RP2040.

Reply:
We are aware of what the define does. The question is "I need to use PIO1 instead, because....?". The answer to that is you dont as you have nothing else that depends on PIO so its fine to use the defaults and the line needs to be removed.

Reply:
I understand now. These details need to be clarified in the QMK documentation, specifically in the WS2812 PIO driver section. When I first created the config files as a beginner in QMK, I assumed that PIO0 was being used by another function of the RP2040 MCU. So I thought that’s why there is an option to use PIO1 but I was wrong.